maps/aurora/aurora-4_mainlevel.dmm

```diff
@@ -59130,7 +59130,8 @@
 /obj/effect/decal/cleanable/dirt,
 /obj/effect/landmark/costume/imperium_monk,
 /obj/structure/closet{
-	icon_state = "cabinet"
+	icon_state = "cabinet";
```

Comment:
I think these var changes should be done in a subtype rather than manually editing every instance when mapped.

Reply:
Also possibly already resolved.